pkg/cloud/services/compute/instance.go

```diff
@@ -265,10 +266,15 @@ func createInstance(is *Service, clusterName string, i *infrav1.Instance) (*infr
 
```

Comment:
In line 221 we could add a metric for `trunks.Create`. And in line 629 for `trunks.Delete`.